shared/haml/user_header.haml

```diff
@@ -1,12 +1,14 @@
--# Locals: current_user, loc_prefix, [session_pairings]
+-# Locals: current_user, loc_prefix, [session_pairings], level
 :ruby
   user_under_13 = request.cookies["_user_type_#{rack_env}"] == "student_y"
+  level = defined?(local_assigns).nil? ?
```

Comment:
Can you just use `local_assigns.has_key? :level`?

Reply:
If I do that I end up with `undefined local variable or method `local_assigns'` on Pegasus because the user_header_option I'm interested in is coming from the _header file defined in dashboard. There might be a cleaner way than what I have, but I basically want to say "If local_assigns is defined (because we're on dashboard) then use the level value, if not then there's definitely not a level so show the create dropdown button".

Reply:
Ah, I see.  You are pioneering here!  What's the precedence between `defined?` and `.nil?`?

Reply:
I think what I had was overkill.

Reply:
Oh, I mean which of those two pieces of code is executed first?